sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/ContentSource.java

```diff
@@ -4,24 +4,24 @@
 package com.azure.ai.formrecognizer.documentanalysis.administration.models;
 
 /**
- * Class representing the source of the training content.
+ * Class representing the source of the data/content.
  */
-public abstract class TrainingDataContentSource {
-    private final TrainingDataContentSourceKind kind;
+public abstract class ContentSource {
```

Comment:
TrainingDataContentSource was more descriptive. Did all languages switch to using `ContentSource` instead?

Reply:
yes, talking with the service team, it seems that this type would be used across analysis API's too so making it more generic.